plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestBrokerQueries.java

```diff
@@ -46,6 +46,7 @@
     private static final DataSchema DATA_SCHEMA;
     private static final List<Object[]> TEST_DATA;
     private static final ResultTable RESULT_TABLE;
+    private static final int LIMIT_FOR_BROKER_QUERIES = 2;
```

Comment:
Looks like TestBrokerQueries could benefit from a test that shows that the broker queries limit gets applied.